Resources/public/scss/modules/universal-discovery/_content.table.header.scss

```diff
@@ -2,14 +2,14 @@
     &__list-headers {
         display: flex;
         flex-wrap: nowrap;
+        background-color: $ez-white;
```

Comment:
Note: moving this style here from `.c-content-table-header__list-header` fixes this grey lines (hard to spot):
![screen shot 2019-01-18 at 16 50 05](https://user-images.githubusercontent.com/10233057/51397281-27703500-1b41-11e9-93a1-7db36348f8ef.png)

closer:
<img src="https://user-images.githubusercontent.com/10233057/51397309-3656e780-1b41-11e9-8895-6b549139cea7.png" width="200">

CLOSER: 😄
<img src="https://user-images.githubusercontent.com/10233057/51397319-39ea6e80-1b41-11e9-956f-2e8062ab1521.png" width="100">

Reply:
Fix moved to other PR: https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/135